endpoints/setuid.go

```diff
@@ -101,6 +103,21 @@ func NewSetUIDEndpoint(cfg *config.Configuration, syncersByBidder map[string]use
 			return
 		}
 
+		activities, activitiesErr := privacy.NewActivityControl(account.Privacy)
+		if activitiesErr != nil {
+			if errortypes.ContainsFatalError([]error{activitiesErr}) {
+				w.WriteHeader(http.StatusBadRequest)
+				return
```

Comment:
We should probably record a metric and update the analytics object `so` here as well.
Should the param passed to `metricsEngine.RecordSetUid` be one of the account options (`metrics.SetUidAccountConfigMalformed` or `SetUidAccountInvalid`) or something more generic?

Reply:
This is good point. We discussed this offline and decided to add metrics in a separate PR.